src/ProfileManager.ts

```diff
@@ -104,7 +104,7 @@ export class ProfileManager implements IProfileManager {
 
       this.logger.debug(`retrieve CIRA MPS Password for cira config ${ciraConfig.configName}`)
       if (this.configurator?.secretsManager) {
-        ciraConfig.password = await this.configurator.secretsManager.getSecretFromKey(`${EnvReader.GlobalEnvConfig.VaultConfig.SecretsPath}CIRAConfigs/${ciraConfig.configName}`, `${ciraConfig.configName}_CIRA_PROFILE_PASSWORD`)
+        ciraConfig.password = await this.configurator.secretsManager.getSecretFromKey(`${EnvReader.GlobalEnvConfig.VaultConfig.SecretsPath}CIRAConfigs/${ciraConfig.configName}`, 'MPS_PASSWORD')
```

Comment:
I kinda liked that this was called CIRA_PROFILE_PASSWORD, what was the motivation to change it?

Reply:
Its actually mps username and mps password, so changed it accordingly.